src/adapter.js

```diff
@@ -144,10 +144,8 @@ const adapter = {
                 let callbacks = map.callbacks
 
                 if (callbacks[keypath]) {
-                  callbacks[keypath].slice().forEach(callback => {
-                    if (callbacks[keypath].indexOf(callback) > -1) {
-                      callback()
-                    }
+                  callbacks[keypath].forEach(cb => {
+                      cb()
```

Comment:
Sorry to be petty but I would find this easier to read as `callbacks[keypath].forEach(cb => cb())`, reckon you could change it?